CMakeLists.txt

```diff
@@ -67,6 +67,11 @@ if (F3D_USE_EXTERNAL_NLOHMANN_JSON)
   find_package(nlohmann_json REQUIRED)
 endif ()
 
+if (F3D_PLUGIN_BUILD_VDB)
```

Comment:
@Meakk please note the `F3D_PLUGIN_BUILD_VDB`  option has not beed declared yet. Let me know what you think .

Reply:
VTK does not run the `find_package` itself?

Reply:
it does in the source but not in vtk-config.cmake. I dont think it should.
If someone uses IOOpenVDB target, they have to provide openvdb::openvdb target

Reply:
Is it possible to test if VTK IOVDB module is enabled instead?

Reply:
Thats not what you want to test, the module can be enabled but OpenVDB may not be available.

Reply:
switched to looking for the VTK module optionally instead